connectivity/drivers/802.15.4_RF/mcr20a-rf-driver/source/NanostackRfPhyMcr20a.cpp

```diff
@@ -140,7 +140,7 @@ static const uint8_t  pll_int[16] =  {0x0B,   0x0B,   0x0B,   0x0B,   0x0B,   0x
 static const uint16_t pll_frac[16] = {0x2800, 0x5000, 0x7800, 0xA000, 0xC800, 0xF000, 0x1800, 0x4000, 0x6800, 0x9000, 0xB800, 0xE000, 0x0800, 0x3000, 0x5800, 0x8000};
 
 /* Channel configurations for 2.4 */
-static const phy_rf_channel_configuration_s phy_24ghz = {.channel_0_center_frequency = 2405000000U, .channel_spacing = 5000000U, .datarate = 250000U, .number_of_channels = 16U, .modulation = M_OQPSK};
+static const phy_rf_channel_configuration_s phy_24ghz = {2405000000U, 5000000U, 250000U, 16U, M_OQPSK, MODULATION_INDEX_0_5};
```

Comment:
i agree with you but my intetion is to reduce the warning spam when turning on pedandic and wall
the problem is that we are still compiling with C++14
if we want to switch to C++20 there are a lot of other code which could be optimized(eg variadic templates)
but i assume in this case it would be better to omit the warining when compiling with c++14